### DISCUSSION TITLE
How to configure different themes per filetype?

### DISCUSSION THREAD

**Discussion by sgon00** [NvChad/NvChad#3150] [Q&A] [answered]
How to configure different themes per filetype?
I found a question at https://github.com/NvChad/NvChad/discussions/2612. But that question is marked as `Answered`. So I have to re-ask this question.
If it's not possible, where can I send a feature request? Thanks a lot.

**Comment by siduck** [comment, marked as answer]
```lua

hl_add = {

["@comment.lua"] =  { fg = 'red' },
["@comment.python"] = { bg = 'pink' }

}
```

**Reply by sgon00** [reply]
Thanks a lot for your reply. But how to use different built-in themes for different file types? I would like to use `nord` for markdown files and `onedark` for other file types. All other themes are too colorful for markdown files and disturb my attention when reading. Cheers.

**Reply by siduck** [reply]
@sgon00  not possible at the moment

**Reply by sgon00** [reply]
Where can I send a feature request? Some projects allow users to send feature requests in issues, but since NvChad issues are too clean,  it's most likely not suitable for this purpose. Thanks.

**Reply by siduck** [reply]
it is not possible as per base46's architecture. so this feature cant be added sadly but a script can be made :grin:
